packages/renderer/src/lib/pod/PodEmptyScreen.svelte

```diff
@@ -1,33 +1,10 @@
 <script lang="ts">
-import Fa from 'svelte-fa/src/fa.svelte';
-import { faPaste } from '@fortawesome/free-solid-svg-icons';
-import { onMount } from 'svelte';
-
-export let pods: Array<unknown>;
-
-function copyRunInstructionToClipboard() {
-  const text = copyText?.innerText;
-  navigator.clipboard.writeText(text);
-}
-
-let copyText;
-
-onMount(async () => {
-  copyText = document.getElementById('noPodCommandLine') as HTMLElement;
-});
+import EmptyScreen from '../ui/EmptyScreen.svelte';
+import PodIcon from '../images/PodIcon.svelte';
 </script>
 
-<div class="h-full min-w-full flex flex-col" class:hidden="{pods.length > 0}">
-  <div class="pf-c-empty-state h-full">
-    <div class="pf-c-empty-state__content">
-      <i class="fas fa-cubes pf-c-empty-state__icon" aria-hidden="true"></i>
-      <h1 class="pf-c-title pf-m-lg">No pods</h1>
-      <div class="pf-c-empty-state__body">Run a first pod using the following command line:</div>
-      <div class="flex flex-row bg-gray-800 w-full items-center p-2 mt-2">
-        <div id="noPodCommandLine">podman pod create --label myFirstPod</div>
-        <button title="Copy To Clipboard" class="mr-5" on:click="{() => copyRunInstructionToClipboard()}"
-          ><Fa class="ml-3 h-5 w-5 cursor-pointer rounded-full text-3xl text-sky-800" icon="{faPaste}" /></button>
-      </div>
-    </div>
-  </div>
-</div>
+<EmptyScreen
+  icon="{PodIcon}"
+  title="No pods"
+  message="Run a first pod using the following command line:"
+  commandline="podman volume create myFirstVolume" />
```

Comment:
looks like a copy/paste error

Reply:
Fixed in new commit.

A little embarrassed at how many times I looked at that and didn't notice...